package.json

```diff
@@ -58,7 +58,7 @@
   ],
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=20"
+    "node": ">=18"
```

Comment:
I don't think this prevents operations, just causes warnings, so keep it as >=20 to be in line Node LTS lifecycle?